common/djangoapps/util/views.py

```diff
@@ -249,10 +250,17 @@ def _record_feedback_in_zendesk(
     """
     zendesk_api = _ZendeskApi()
 
-    additional_info_string = (
-        u"Additional information:\n\n" +
-        u"\n".join(u"%s: %s" % (key, value) for (key, value) in additional_info.items() if value is not None)
-    )
+    # Extract information that can be stored in custom fields from the additional_info dictionary.
```

Comment:
Extract custom_field info from the additional_info dictionary. This allows us to fallback and include the custom field information even when custom fields have not been configured.